### PR TITLE
Framework: avoid assign null to this.bindStateValue

### PR DESCRIPTION
**Pull request by scottoasis**
R= @yorkie @zhangweijie-cn

### CODE REVIEW

**Comment by yorkie** [comment]
LGTM

File: src/select/index.jsx [hunk 1]
```diff
@@ -24,7 +24,14 @@ var Select = UIFramework.Component(class extends UIFramework.Control {
     },
   };
   get defaultValue() {
-    return this.props.options[0].value;
+    let defaultValue;
```
Comment (yorkie) [inline src/select/index.jsx]: `defaultValue` here will be confused, suggestion: `val`.

File: src/select/index.jsx [hunk 2]
```diff
@@ -24,7 +24,14 @@ var Select = UIFramework.Component(class extends UIFramework.Control {
...
+    try {
+      defaultValue = this.props.options[0].value;
+    } catch (error) {
+      return null;
```
Comment (yorkie) [inline src/select/index.jsx:31]: `val = null` because we did the `return` at `finally` block?

**Comment by yorkie** [comment]
LGTM

File: src/select/index.jsx (hunk 2 above)
Comment (scottoasis) [inline src/select/index.jsx:31]: Returning a instant value is more preferred here as by doing so, we avoid unnecessary state/variable mutation.

Comment (yorkie) [inline src/select/index.jsx:31]: Ok, got it, LGTM now :-)

**Pull request merged**